docs/README.md

```diff
@@ -0,0 +1,20 @@
+# Ace Documentation
+
+This directory contains documentation for `ace`. If you are interested in contributing new documentation, please read the [contributor guidelines](../.github/CONTRIBUTING.md) and [What Nobody Tells You About Documentation](https://documentation.divio.com) for help getting started.
+
+## API
+
+The API is a low-level, type-safe, straightforward binding to Ace:
+
+* All effectful JavaScript functions (i.e. virtually *all* functions!) are wrapped in the `Eff` monad for the appropriate effect type (`random`, `dom`, etc.).
+* All JavaScript parameter or return values which may be nullable are wrapped / unwrapped using `Maybe`.
+* PureScript record types are used for JavaScript objects.
+* JavaScript's fake sum types are mapped to PureScript's real sum types, which necessitates encoding / decoding.
+
+## Examples
+
+The `examples` directory contains an `index.html`, which contains a variety of examples of how to embed the Ace editor and configure it using the PureScript API.
+
+## Tests
+
+This project has no real tests. However, most bound functions are exercised in the examples, to ensure the bindings and necessary wrapping, unwrapping, encoding, and decoding has been performed correctly.
```

Comment:
Not sure if these should be split up into separate docs sections.

Reply:
I think that these can be condensed and put in the `Documentation` section of the main README -- the section where it states where you can find different forms of documentation. I suggest:

1. Leave the API section as-is (though it ought to be improved over time)
2. Remove the Examples section here and add it as a new bullet point to the documentation section of the repository README (for example: `* The [examples directory](./examples) contains an HTML file demonstrating how to embed the Ace editor and configure it via PureScript`)
3. Remove the Tests section (users of the library don't really need to know this)

Reply:
I think we can leave the API section as-is, remove the Examples section in favor of a simple bullet point in the main repo README (`* The [examples directory](./examples) demonstrates how to embed the Ace editor and configure it via the PureScript API`), and remove the tests section.